x-pack/plugins/ml/public/application/explorer/anomaly_timeline_state_service.ts

```diff
@@ -731,8 +732,7 @@ export class AnomalyTimelineStateService extends StateService {
 
   public getSwimLaneBucketInterval$(): Observable<TimeBucketsInterval> {
     return this._swimLaneBucketInterval$.pipe(
-      // @ts-ignore
-      skipWhile((v) => !v),
+      filter((v): v is TimeBucketsInterval => !v),
```

Comment:
this change is not related to the PR, just wanted to get rid of the `@ts-ignore`